mix.exs

```diff
@@ -93,4 +94,13 @@ defmodule ArchEthic.MixProject do
       {:knigge, "~> 1.4"}
     ]
   end
+
+  defp aliases do
+    [
+      "dev.setup": ["deps.get", "cmd npm install --prefix assets"],
```

Comment:
Good one I think 👍 
But a dedicated PR for improvements for aliases would be better, so anyone could review it.